### ISSUE TITLE
NO MORE API

### ISSUE THREAD

**Issue by xCONFLiCTiONx** [evemondevteam/evemon#90]
So, are you guys still maintaining this application? It would be great if it had the ESI system since EVE no longer uses API's. If no one is please someone take over or something. This program is great.

**Comment by aydjay** [comment]
https://github.com/peterhaneve/evemon/

This is a fork that is working towards ESI compliance

**Comment by xCONFLiCTiONx** [comment]
DUDE! You are awesome!

**Comment by xCONFLiCTiONx** [comment]
Sorry but how do you get it to work? 

{"error":"invalid_request","error_description":"the callback uri doesn't match the value stored for this client"}

I think it might be my callback uri. I don't know what to put there.

**Comment by ABespalov** [comment]
I have same troubles.
The NetworkConstants.resx have next lines:
```
  <data name="SSORedirect" xml:space="preserve">
    <value>http://localhost:{0:D}/callback</value>
  </data>
```
but what about port number?

**Comment by Feoni4** [comment]
PORT = 4916 - const in SSOWebServer

**Comment by xCONFLiCTiONx** [comment]
Yep, works like a champ. You don't need to open the ports in firewall just make the application at https://developers.eveonline.com and then make sure that the Callback URL is:
http://localhost:4916/callback

Thank you!

**Comment by Nexaryon** [comment]
Getting
{"error":"invalid_scope","error_description":"The requested scopes either don't exist, or are not valid for this client"}
what are all the needed scopes?

**Comment by UweSchneider** [comment]
Scope extracted from URL:

esi-alliances.read_contacts.v1

esi-assets.read_assets.v1
esi-assets.read_corporation_assets.v1

esi-calendar.read_calendar_events.v1

esi-characters.read_agents_research.v1
esi-characters.read_blueprints.v1
esi-characters.read_contacts.v1
esi-characters.read_corporation_roles.v1
esi-characters.read_fatigue.v1
esi-characters.read_fw_stats.v1
esi-characters.read_loyalty.v1
esi-characters.read_medals.v1
esi-characters.read_notifications.v1
esi-characters.read_standings.v1

esi-clones.read_clones.v1
esi-clones.read_implants.v1

esi-contracts.read_character_contracts.v1
esi-contracts.read_corporation_contracts.v1

esi-corporations.read_blueprints.v1
esi-corporations.read_contacts.v1
esi-corporations.read_divisions.v1
esi-corporations.read_fw_stats.v1
esi-corporations.read_medals.v1
esi-corporations.read_outposts.v1
esi-corporations.read_standings.v1
esi-corporations.read_structures.v1

esi-industry.read_character_jobs.v1
esi-industry.read_corporation_jobs.v1

esi-killmails.read_corporation_killmails.v1
esi-killmails.read_killmails.v1

esi-location.read_location.v1
esi-location.read_ship_type.v1

esi-mail.read_mail.v1

esi-markets.read_character_orders.v1
esi-markets.read_corporation_orders.v1
esi-markets.structure_markets.v1

esi-planets.manage_planets.v1

esi-skills.read_skillqueue.v1
esi-skills.read_skills.v1

esi-universe.read_structures.v1

esi-wallet.read_character_wallet.v1
esi-wallet.read_corporation_wallets.v1

**Comment by Nexaryon** [comment]
@UweSchneider tyvm worsk now o/

**Comment by xCONFLiCTiONx** [comment]
I just clicked on all of the scopes in the Application I made on the eve online dev website and it works fine. Way faster to do that then to actually skim through that massive list looking for all of those.

**Comment by dance** [comment]
Works perfectly, thank you very much! By the way what about people having alpha accounts only? They cannot manage applications ("Your EVE Online account is not valid for the purpose of registering as a developer."). Where do they get Client ID and Client Secret for evemon from?

**Comment by Dizirgee** [comment]
It would be really nice to have standalone/zip version instead of installer
